toplevel/search.ml

```diff
@@ -98,11 +98,13 @@ let generic_search glnumopt fn =
   iter_declarations fn
 
 (** Standard display *)
-
 let plain_display accu ref env c =
-  let pc = pr_lconstr_env env Evd.empty c in
   let pr = pr_global ref in
-  accu := hov 2 (pr ++ str":" ++ spc () ++ pc) :: !accu
+  if !Constrextern.search_minimal then
+    accu := pr :: !accu
+  else
```

Comment:
you should always use `begin .. end` around a multi-line `else`-branch of type `unit`.